patch_geopandas.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# manually edit geopandas/label/dev to match current pandas deployment
+# see https://github.com/geopandas/geopandas/issues/473#issuecomment-465060849
+# should be removed if geopandas core becomes vectorized, whether the
+# cythonized branch gets merged in or the geometry calculations are handled
+# by pyGEOS, either through shapely or replacing shapely. See
+# https://github.com/Toblerity/Shapely/issues/782.
```

Comment:
@bolliger32 I added this comment to patch_geopandas.sh to remind ourselves what's going on here. if this looks good to you go ahead and merge! 👍